src/Adapter/Image/ImagePathFactory.php

```diff
@@ -45,12 +45,12 @@ public function __construct(
     }
 
     /**
-     * @param int $entityId
+     * @param int|string $imageName
      *
      * @return string
      */
-    public function getPath(int $entityId): string
+    public function getPath(int|string $imageName): string
```

Comment:
@Codencode I would remove the `int|string` here and it will work

Reply:
Yes, I was looking into the issue.
Ok, I'll remove it.
Thanks!